python/observability-online-eval/pydantic-ai/advanced_agent.ipynb

```
@@ -0,0 +1,239 @@
+   "source": [
+    "### Create an Advanced Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "1b2565c4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_advanced_agent():\n",
```

Ruff (0.12.2)

`await` statement outside of a function

(F704)